rfc/text/advanced/ap_09_rpc_progressive_call_results.md

```diff
@@ -339,3 +339,8 @@ Note: Any `ERROR` returned by the *Callee*, in response to the `INTERRUPT`, is i
 A *Callee* that does not support progressive results SHOULD ignore any `INVOCATION.Details.receive_progress` flag.
 
 A *Callee* that supports progressive results, but does not support call canceling is considered by the *Dealer* to not support progressive results.
+
+
+**Timeouts**
+        
+When the *Call Timeouts* feature is used in combination with Progressive Call Results, the `CALL.Options.timeout|int` option shall represent the time limit between the initial call and the first result, and between results thereafter.
```

Comment:
does that mean, given a call timeout T:

* `t(YIELD_1 or ERROR_1 received from callee) - t(CALL received from caller) < T AND`
* `t(YIELD_n+1 or ERROR_n+1 received from callee) - t(YIELD_n received from callee) < T`

?

Reply:
I understand it like you wrote

Reply:
@oberstet Assuming that the timeout is handled by the Dealer (and not delegated to the Callee), then yes, that's what I meant. There should be no need to add any timestamps to messages.

Would you like me to add your equations so that it's more formal?

Reply:
> Would you like me to add your equations so that it's more formal?

for my taste, yes, something a bit more formal or descriptive. e.g.

* this is about dealer-side/enforced callee timeouts
* the timeout applies for "time between successive progressive results" (_not_ a timeout for the complete call from start to last final result)
* a consequence of the last: the dealer needs to track 1 timeout instance per received progressive result